src/components/header/WomenDropdown.jsx

```diff
@@ -0,0 +1,355 @@
+        width: '185px',
+      }}
+    />
+  );
+}
+
+export default function WomenDropdown() {
+  const classes = useStyles();
+  const [dropDownOpen, setDropDownOpen] = React.useState(null);
+  const open = Boolean(dropDownOpen);
+
+  // Function for the dropdown menus to show when clicked
+  const handleClick = (event) => {
+    setDropDownOpen(event.currentTarget);
+  };
+
+  // Function for the dropdown menu to close when not needed
+  const handleClose = () => {
+    setDropDownOpen(null);
+  };
+
+  return (
+    <>
+      <Box sx={{ display: 'flex', alignItems: 'center', textAlign: 'center' }}>
+        <Tooltip>
+          <IconButton
+            onClick={handleClick}
+            className="navItems"
+            onMouseOver={handleClick}
+            size="small"
+            sx={{ ml: 2 }}
+            aria-controls={open ? 'account-menu' : undefined}
+            aria-haspopup="true"
+            aria-expanded={open ? 'true' : 'false'}
+          >
+            <div className={`${classes.catContainerWomen}`}>
+              <span>
+                <b>WOMEN</b>
+              </span>
+            </div>
+          </IconButton>
+        </Tooltip>
+      </Box>
+      <Menu
+        anchorEl={dropDownOpen}
+        id="account-menu"
+        open={open}
+        onClose={handleClose}
+        onClick={handleClose}
+        onMouseLeave={handleClose}
+        PaperProps={{
+          elevation: 0,
+          sx: {
+            overflow: 'visible',
+            filter: 'drop-shadow(0px 2px 8px rgba(0,0,0,0.32))',
+            mt: 1.5,
+            '& .MuiAvatar-root': {
+              width: 32,
+              height: 32,
+              ml: -0.5,
+              mr: 1,
+            },
+            '&:before': {
+              content: '""',
+              display: 'block',
+              position: 'absolute',
+              top: 0,
+              right: 14,
+              width: 10,
+              height: 10,
+              bgcolor: 'background.paper',
+              transparency: '50%',
+              zIndex: 0,
+            },
+            '&:hover': {
+              color: 'black',
+            },
+          },
+        }}
+      >
+        <Grid container>
+          <Grid item>
+            <div
+              style={{
+                backgroundColor: '#fff',
+                padding: '6px',
+                height: '440px',
+                width: '220px',
+              }}
+            >
+              <Heading>
+                <b>Indian & Fusion Wear </b>
+              </Heading>
+              <MenuBold>Kurtas & Suits</MenuBold>
+              <MenuBold>Kurtis, Tunics & Tops</MenuBold>
+              <MenuBold>Ethnic Wear</MenuBold>
+              <MenuBold>Leggings, Salwars & Churidars</MenuBold>
+              <MenuBold>Skirts & Palazzos</MenuBold>
+              <MenuBold>Sarees</MenuBold>
+              <MenuBold>Dress Materials</MenuBold>
+              <MenuBold>Lehenga Cholis</MenuBold>
```

Comment:
For same type of components with different text data make an array constants and loop through its length and pass the current index data to that HTML Dom.